lib/services/elasticsearch.js

```diff
@@ -982,7 +986,8 @@ function getElasticsearchRequest(request, kuzzle) {
   }
 
   ['from', 'size', 'scroll', 'scrollId', 'refresh'].forEach(argumentName => {
-    if (typeof request.input.args[argumentName] !== 'undefined') {
+    if (typeof request.input.args[argumentName] !== 'undefined' &&
+        extraParams.includes(argumentName)) {
```

Comment:
But it will break my unit tests so I hate you. But your solution is good. But it will break my unit tests... (put this into a `while (1)`)